Bwt/BwtProj/Bwt.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace BWT;
+
+/// <summary>
+/// Burrows Wheeler Transformation implementation and inversion of it
+/// </summary>
+public static class Bwt
+{
+    private const int charAmount = 256;
+    
+    /// <summary>
+    /// BWT based on building <see cref="SuffixArray"/> 
+    /// </summary>
+    public static Tuple<char[], int> BwTransformation(String inputString)
```

Comment:
```suggestion
    public static (char[], int) BwTransformation(String inputString)
```
Всё же Tuple не используются, если нет веских причин их использовать. Тут --- нет, тут возвращается 32-битное целое и указатель. Копировать на кучу и возвращать указатель на них --- это сильный удар по скорости работы.